xdsl/dialects/scf.py

```diff
@@ -536,6 +536,45 @@ def __init__(
     def get(cond: SSAValue | Operation, *output_ops: SSAValue | Operation) -> Condition:
         return Condition(cond, *output_ops)
 
+    def print(self, printer: Printer):
+        printer.print("(", self.cond, ") ")
+        if self.attributes:
+            printer.print_op_attributes(self.attributes)
+            printer.print(" ")
```

Comment:
I'm pretty sure that this will print two spaces when printing the op attributes.
Also, it is missing a test I think?

Reply:
it was printing an extra space before, not after, I updated the method